### ISSUE TITLE
geoip2 suddenly giving syntax error

### ISSUE THREAD

**Issue by mlevin2** [YOURLS/YOURLS#2591]
After years of working perfectly, today I started noticing problems.

When hitting a short link from my desktop browser, it works fine, but if I hit the link from the gmail app on my Android phone, I get a 500 error and this in the logs:

```
[Fri Jan 03 16:14:57 2020] [error] [client xx.xx.xx.xx] PHP Parse error:  syntax error, unexpected '[' in /var/sites/xxxxxxx/includes/vendor/geoip2/geoip2/src/Database/Reader.php on line 51
```

That line is:

```php
$locales = ['en']
```

I worked around it temporarily by commenting out the call to geoip2 in `functions.php`:

```php
'location' => "" /*yourls_geo_ip_to_countrycode($ip)*/,
```

Any ideas?

**Comment by LeoColomb** [comment]
Hello @mlevin2, thanks for this report.

Please provide technical context of your installation (php version, mobile app, yourls version, etc.)

**Comment by mlevin2** [comment]
PHP version: PHP 5.3.10-1ubuntu3.26 with Suhosin-Patch (cli) (built: Feb 13 2017 20:37:53) _yes, I know that's really old - I've been meaning to rebuild this box :-)_

Yourls version: YOURLS v 1.7.4

The failure was happening when I clicked a link from the latest version of the Android Gmail app on Android 10. When I clicked the same link from the latest version of Chrome on Catalina, no geoip error. Both devices were on the same wifi network at the time. Behavior was consistent - click on desktop, works fine, click the same link from the phone, and I would get that error.
